application/features/email.feature

```diff
@@ -185,7 +185,6 @@ Feature: Email
       | to send                 | does exist     | does NOT exist  | delete_mfa        | should      |
       | to send                 | does NOT exist | does exist      | delete_mfa        | should      |
       | to send                 | does exist     | does exist      | delete_mfa        | should      |
-      | to send                 | does NOT exist | does NOT exist  | delete_mfa        | should NOT  |
```

Comment:
Out of curiosity, why was this example removed from the test?

Reply:
It was moved down to a separate test, because this test doesn't have a way of saying that  the Mfa used to exist but doesn't anymore.  It just says that right now, it doesn't exist.  So, it didn't have access to a Mfa instance to send to the Emailer method.